py/server/deephaven/experimental/s3.py

```diff
@@ -45,9 +131,7 @@ def __init__(self,
                  num_concurrent_write_parts: Optional[int] = None,
                  profile_name: Optional[str] = None,
                  config_file_path: Optional[str] = None,
-                 credentials_file_path: Optional[str] = None,
-                 use_profile_credentials: bool = False,
-                 use_default_credentials: bool = False):
+                 credentials_file_path: Optional[str] = None):
```

Comment:
Why is there still a non-deprecated `credentials_file_path` if we now have `credentials`?

Reply:
That is different from `credentials`. A credentials file contains a lot more stuff rather than just credentials, like region. That is why even though this file is used for credentials too, it's specified in instructions.